wire/value.go

```diff
@@ -53,108 +47,110 @@ func (v *Value) Type() Type {
 func (v *Value) Get() interface{} {
 	switch v.typ {
 	case TBool:
-		return v.tbool
+		return v.GetBool()
 	case TI8:
-		return v.ti8
+		return v.GetI8()
 	case TDouble:
-		return v.tdouble
+		return v.GetDouble()
 	case TI16:
-		return v.ti16
+		return v.GetI16()
 	case TI32:
-		return v.ti32
+		return v.GetI32()
 	case TI64:
-		return v.ti64
+		return v.GetI64()
 	case TBinary:
-		return v.tbinary
+		return v.GetBinary()
 	case TStruct:
-		return v.tstruct
+		return v.GetStruct()
 	case TMap:
-		return v.tmap
-	case TSet:
-		return v.tset
-	case TList:
-		return v.tlist
+		return v.GetMap()
+	case TSet, TList:
+		return v.GetList()
 	default:
 		panic(fmt.Sprintf("Unknown value type %v", v.typ))
 	}
 }
 
 // NewValueBool constructs a new Value that contains a boolean.
 func NewValueBool(v bool) Value {
+	var n uint64 = 0
```

Comment:
nit: can drop the `= 0` since it's the default anyway. or if you want the 0 to be explicit, `n := uint64(0)`